Week2/homework/js-exercises/convertGrades.js

```diff
@@ -0,0 +1,35 @@
+'use strict';
+
+let gradeUsa = '';
+
+let score ='';
+//Created this if else function to compare score with different cases to find out USA equivelent of our score.
+function gradeCalculator(score) {
+    if (90 <= score && score <= 100) {
+    gradeUsa = 'A';
+    }
+        else if (80 <= score && score <= 89) {
```

Comment:
best to use < 90 on the upper bound, as with your logic, a number like `89.95` would not match any condition!